clients/python/build/.openapi-generator-ignore

```diff
@@ -6,6 +6,14 @@ README.md
 pyproject.toml
 setup.py
 
+## Do not generate irrelevant files
+requirements.txt
+test-requirements.txt
```

Comment:
These 2 are simply just confusing for users. If they install locally from source (not common, but can happen) they may be confused about the dependencies that OpenAPI declares are necessary.